lua/treesitter-context.lua

```diff
@@ -392,31 +392,27 @@ end
 
 --- @generic F: function
 --- @param fn F
+--- @param ms? number
 --- @return F
-local function throttle_fn(fn)
-  local recalc_after_cooldown = false
-  local cooling_down = false
-  local function wrapped()
-    if cooling_down then
-      recalc_after_cooldown = true
+local function debounce(fn, ms)
+  ms = ms or 20
+  fn = vim.schedule_wrap(fn)
```

Comment:
Could we wrap this later so if it's the first call and is executed immediately, we don't have any delay? I'm still seeing just a bit of delay and this change removes it

Reply:
done